cmd/nerdctl/network_prune.go

```diff
@@ -97,7 +97,7 @@ func networkPrune(cmd *cobra.Command, client *containerd.Client, ctx context.Con
 
 	var removedNetworks []string // nolint: prealloc
 	for _, net := range e.Networks {
-		if net.Name == "host" || net.Name == "none" {
+		if net.Name == "host" || net.Name == "none" || net.Name == "bridge" {
```

Comment:
Can you please refactor move all of these to a constant and look them over so that we do not expand this list